tests/fps_tests.py

```diff
@@ -1273,6 +1273,69 @@ def test_expected_case(self, mock_open_and_load_json):
         loaded_sets = fp.load_sets()
         fp.find_invalid_well_known(loaded_sets)
         self.assertEqual(fp.error_list, [])
+    @mock.patch('FpsCheck.FpsCheck.open_and_load_json', 
```

Comment:
Missing blank line between this test and the previous one. Same comment for the other new tests.